### PR TITLE
Updates README "Getting involved".

### PR DESCRIPTION
**Pull request by andsteing**
The concommitant changes to the `.sh` files make sure that
`run_all_tests.sh` fails if `svn` is not installed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -222,6 +222,18 @@ When working with large-scale input data, it is important to create large enough
 TODO: Add an example for running on Google Cloud.
 
 ## Getting involved
+
+Currently, you need to install Python 3.6 and `svn` for developing Flax. After installing these prerequisites, you can clone the repository, set up your local environement, and run all tests with the following commands:
```
Comment (marcvanzee) [inline README.md:226]:
```suggestion
Currently, you need to install Python 3.6 and `svn` for developing Flax. After installing these prerequisites, you can clone the repository, set up your local environment, and run all tests with the following commands:
```

Comment (marcvanzee) [inline README.md:226]: Sorry for the silly question, but why do we need svn?

**Review by marcvanzee — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (andsteing) [reply README.md:226]: For downloading tfds metadata (a directory inside the tensorflow/datasets repo).

Using git for downloading the directory would require us to clone the entire tensorflow/datasets repo (currently ~226M when cloning with `--depth=1`).

**Comment by marcvanzee** [comment]
Some tests are failing, could you check if this is related to your `-e` change?

**Review by marcvanzee — CHANGES_REQUESTED** [review]
Some tests are failing, could you check if this is related to your `-e` change?

File: README.md (hunk 1 above)
Comment (andsteing) [reply README.md]: huh - I thought I had a spell checker installed ?!
apparently not. installed one just now :)

**Comment by andsteing** [comment]
Fixed failing test, PTAL.

File: README.md (hunk 1 above)
Comment (marcvanzee) [reply README.md]: Hmm interesting. In my VMs I never install svn and everything seems to work fine. Are you sure this is needed for all Flax development, or only if you want to use TFDS? (which is not a requirement for developing with Flax, I think)

Comment (andsteing) [reply README.md]: It's for running the examples.
If your VMs succeeded the `./tests/run_all_test.sh` then they either had a `.tfds` directory in the root or `svn` was installed on them by default.

File: README.md [hunk 2]
```diff
@@ -222,6 +222,18 @@ When working with large-scale input data, it is important to create large enough
 TODO: Add an example for running on Google Cloud.
 
 ## Getting involved
+
+Currently, you need to install Python 3.6 and `svn` for developing Flax. After installing these prerequisites, you can clone the repository, set up your local environment, and run all tests with the following commands:
```
Comment (marcvanzee) [inline README.md:226]:
```suggestion
Currently, you need to install Python 3.6 for developing Flax, and `svn` for running the `run_all_tests.sh` script. After installing these prerequisites, you can clone the repository, set up your local environment, and run all tests with the following commands:
```

**Review by marcvanzee — APPROVED** [review]

**Pull request merged**